### DISCUSSION TITLE
Show warning when user does FOTA is poor network conditions

### DISCUSSION THREAD

**Discussion by balaji-srin** [hello-nrfcloud/web#914] [Ideas]
It would be great if the user gets a warning or a info that it is not recommended to do a FOTA due to (last known) network conditions.

**Comment by coderbyheart** [comment]
Can you explain how that can be determined?

**Comment by martiles** [comment]
Assume this can be solved by the data gathered from the connection evaluation feature. -->https://docs.nordicsemi.com/bundle/nwp_043/page/WP/nwp_043/intro.html

**Comment by coderbyheart** [comment]
So, if device has reported Energy estimate value of 5 or 6 a warning should be displayed?

**Reply by martiles** [reply]
That would be informative for the user, yes

**Comment by coderbyheart** [comment]
This is not possible, because the firmware stopped sending signal quality to the cloud:

https://github.com/hello-nrfcloud/firmware/issues/499
